src/Apps/Consign/Routes/SubmissionFlow/ThankYou/__tests__/ThankYouWhenFromMyCollection.jest.tsx

```diff
@@ -1,7 +1,5 @@
 import { mount } from "enzyme"
-import { ThankYouWhenFromMyCollection } from "../ThankYouWhenFromMyCollection"
-
-jest.mock("System/useSystemContext")
+import { ThankYouWhenFromMyCollection } from "Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYouWhenFromMyCollection" // pragma: allowlist secret
```

Comment:
**Question:** what's `pragma: allowlist secret`? 👀

Reply:
That's the sing that tells to detect-secrets that it is not a secret
The test was failing here, I had to add the comment for detect-secrets to skip this line